learningresources/tests/test_api.py

```diff
@@ -6,11 +6,14 @@
 
 import logging
 
-from learningresources.models import Course, LearningResource
 from learningresources.api import (
-    create_course, get_resource, get_repo,
+    get_repo,
     NotFound, PermissionDenied,
 )
+from django.db.utils import IntegrityError
```

Comment:
Minor: The Django import should be above those within the lore project. The order is:

* standard library
* pip-installed packages
* local code